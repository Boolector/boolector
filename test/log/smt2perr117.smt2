(assert ((_ zero_extend 2147483640)(_ bv255 8)))
