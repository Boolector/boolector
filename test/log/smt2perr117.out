log/smt2perr117.smt2:1:10: resulting bit-width of 'zero_extend' too large
